### PR TITLE
Feature/url signature length config

### PR DESCRIPTION
**Pull request by nitzanj**

### CODE REVIEW

File: cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractApiTest.java [hunk 1]
```diff
@@ -181,10 +181,11 @@ public void test03ResourcesCursor() throws Exception {
     @Test
     public void test04ResourcesByType() throws Exception {
         // should allow listing resources by type
-        Map resource = preloadResource(ObjectUtils.asMap("tags", UPLOAD_TAGS));
-        Map result = api.resources(ObjectUtils.asMap("type", "upload", "max_results", 500));
+        Map result = api.resources(ObjectUtils.asMap("type", "upload", "max_results", 10));
```
Comment (asisayag2) [inline cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractApiTest.java]: @nitzanj , is that related to the current change?

File: cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractFoldersApiTest.java [hunk 2]
```diff
@@ -37,7 +37,7 @@ public void testRootFolderWithParams() throws Exception {
         String rootFolder2Name = "rootFolderWithParamsTest2" + SUFFIX;
         assertTrue((Boolean) api.createFolder(rootFolder2Name, null).get("success"));
 
-        Thread.sleep(500);
+        Thread.sleep(2000);
```
Comment (asisayag2) [inline cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractFoldersApiTest.java]: @nitzanj , same here. Relevant for the current change?

File: cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractApiTest.java (hunk 1 above)
Comment (nitzanj) [reply cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractApiTest.java]: Not really, but through this PR I found out there was a reliability issue with timing and resource-flooded test account. 
However the test fixes are on their own commit so the history will remain clean, on master too.

File: cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractFoldersApiTest.java (hunk 2 above)
Comment (nitzanj) [reply cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractFoldersApiTest.java]: See above.

File: cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractApiTest.java (hunk 1 above)
Comment (asisayag2) [reply cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractApiTest.java]: Why not having 2 separate PRs?

Comment (nitzanj) [reply cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractApiTest.java]: Quicker and no merges, and it ends up identical on master's history when merged.

Comment (asisayag2) [reply cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractApiTest.java]: Approving, but let's keep PRs as focused as possible in the future.
